src/core_plugins/console/api_server/es_6_0.js

```diff
@@ -0,0 +1,27 @@
+import  _ from 'lodash';
+import Api from './api';
+import { getSpec } from './spec';
+const parts = [
+  require('./es_6_0/aliases'),
+  require('./es_6_0/aggregations'),
+  require('./es_6_0/document'),
+  require('./es_6_0/filter'),
+  require('./es_6_0/globals'),
+  require('./es_6_0/ingest'),
+  require('./es_6_0/mappings'),
+  require('./es_6_0/query'),
+  require('./es_6_0/reindex'),
+  require('./es_6_0/search'),
+];
+
+const ES_6_0 = new Api('es_6_0');
+
+const spec = getSpec();
+Object.keys(spec).forEach(endpoint => {
+  ES_6_0.addEndpointDescription(endpoint, spec[endpoint]);
+});
+_.each(parts, function (apiSection) {
```

Comment:
Is there a reason we don't just `require('./es_6_0/aliases')(ES_6_0)` rather than require into an array first then iterate and call later?

If we can't get rid of the separation, mind swapping out `_.each` with `[].forEach()`?

Reply:
Legacy of old code style.  We can fix in a separate PR to address all of that kind of stuff in one go.